implementations/monix/src/main/scala/sttp/client/impl/monix/MonixWebSockets.scala

```diff
@@ -30,4 +31,28 @@ object MonixWebSockets {
         .guarantee(ws.close())
     }
   }
+  def textAccumulator: (String => WebSocketFrame) => MonixStreams.Pipe[WebSocketFrame, WebSocketFrame] =
```

Comment:
`stringPipe`?

Reply:
technically that is rather a pipe transformation than a pipe

Reply:
`fromTextPipe`?